index.html

```diff
@@ -5,40 +5,263 @@
+                 <ul class="connect_menu">
+                    <li><a href="#"><img src="images/twitter.png"/></a></li>
+                    <li><a href="#"><img src="images/linkin.png"/></a></li>
+                    <li><a href="#"><img src="images/M.png"></a></li>
+                    <li><a href="#"><img src="images/git.png"></a></li>
+                    <li><a href="#"><img src="images/fingers.svg"></a></li>
+                 </ul>
+                            
+                  
+            </div>
+            
+        </section>
+
+
+
+
+
+    <section class="grid_container">
+         <div class="work_item1">
+            <div class="information1">
+                  <img class="portrait1" src="images/Snapshoot Portfolio.png"/>
+             <div class="description1">
+               <h2>
+                <strong class="stronger">Tonic</strong>
+                    
+              </h2>
+            
+                <div class="canopy">
+                    <ul>
+                        <li><div class="canopy1">CANOPY</div></li>
+                     
+                        <li> Full Stack Dev</li>
+                        
+                        <li>2015</li>
+                    </ul>
+                  
+                </div>                
                 <p>
-                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                    A daily selection of privately <br>
+                    personalized reads; no accounts or <br>sign-ups required.
                 </p>
-                <p class="connect">LET'S CONNECT</p>
-             <ul class="connect_menu">
-                <li><a href="#"><img src="images/twitter.png"/></a></li>
-                <li><a href="#"><img src="images/twitter.png"/></a></li>
-                <li><a href="#"><img src="images/M.png"></a></li>
-                <li><a href="#"><img src="images/git.png"></a></li>
-                <li><a href="#"><img src="images/fingers.svg"></a></li>
-             </ul>
+                <div class="language">
+                    <ul>
+                      <li>HTML</li>
+                      <li>CSS</li>
+                      <li>JavaScript</li>
+                    </ul>
+                  </div>
+          
+
+                
+                <div class="project_tag"><button>See project</button></div>
+               
+            </div>
+        </div>
+        
+ </div>
+ <div class="work_item1">
+    <div class="information1">
+          <img class="portrait1" src="images/Snapshoot Portfolio (1).png"/>
+     <div class="description1">
+       <h2>
+        <strong class="stronger"> Multi-Post
+        <br>
+        Stories
+    </strong>
+      </h2>
+    
+        <div class="canopy">
+            <ul>
+                <li><div class="canopy1">CANOPY</div></li>
+             
```

Comment:
- [ ] [Optional] Upon reviewing your code, I noticed that there are several instances of unnecessary whitespace. I would recommend removing the extra spaces to make your code cleaner and more readable. It is important to maintain consistent formatting throughout your codebase. Please take some time to remove the extra spaces. 👍